File: OPENAPI-REST-API/swagger-client/dart/lib/model/inline_response20018_ips.dart
part of swagger.api;

class InlineResponse20018Ips {
  
  double serviceId = null;

  String serviceModule = null;

  String serviceHostname = null;

  InlineResponse20018Ips();

  @override
  String toString() {
    return 'InlineResponse20018Ips[serviceId=$serviceId, serviceModule=$serviceModule, serviceHostname=$serviceHostname, ]';
  }

  InlineResponse20018Ips.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    serviceId = json['service_id'];
    serviceModule = json['service_module'];
    serviceHostname = json['service_hostname'];
  }

  Map<String, dynamic> toJson() {
    return {
      'service_id': serviceId,
      'service_module': serviceModule,
      'service_hostname': serviceHostname
     };
  }

  static List<InlineResponse20018Ips> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineResponse20018Ips>() : json.map((value) => new InlineResponse20018Ips.fromJson(value)).toList();
  }

  static Map<String, InlineResponse20018Ips> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, InlineResponse20018Ips>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new InlineResponse20018Ips.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/java-vertx/src/main/java/io/swagger/server/api/model/InlineResponse20018Ips.java
package io.swagger.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import java.math.BigDecimal;
import io.vertx.core.json.JsonObject;
import io.vertx.codegen.annotations.DataObject;
/**
 * InlineResponse20018Ips
 */




public class InlineResponse20018Ips   {
  private BigDecimal serviceId = null;

  private String serviceModule = null;

  private String serviceHostname = null;

  public InlineResponse20018Ips serviceId(BigDecimal serviceId) {
    this.serviceId = serviceId;
    return this;
  }

  /**
   * Get serviceId
   * @return serviceId
   **/
    public BigDecimal getServiceId() {
    return serviceId;
  }

  public void setServiceId(BigDecimal serviceId) {
    this.serviceId = serviceId;
  }

  public InlineResponse20018Ips serviceModule(String serviceModule) {
    this.serviceModule = serviceModule;
    return this;
  }

  /**
   * Get serviceModule
   * @return serviceModule
   **/
    public String getServiceModule() {
    return serviceModule;
  }

  public void setServiceModule(String serviceModule) {
    this.serviceModule = serviceModule;
  }

  public InlineResponse20018Ips serviceHostname(String serviceHostname) {
    this.serviceHostname = serviceHostname;
    return this;
  }

  /**
   * Get serviceHostname
   * @return serviceHostname
   **/
    public String getServiceHostname() {
    return serviceHostname;
  }

  public void setServiceHostname(String serviceHostname) {
    this.serviceHostname = serviceHostname;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    InlineResponse20018Ips inlineResponse20018Ips = (InlineResponse20018Ips) o;
    return Objects.equals(this.serviceId, inlineResponse20018Ips.serviceId) &&
        Objects.equals(this.serviceModule, inlineResponse20018Ips.serviceModule) &&
        Objects.equals(this.serviceHostname, inlineResponse20018Ips.serviceHostname);
  }

  @Override
  public int hashCode() {
    return Objects.hash(serviceId, serviceModule, serviceHostname);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class InlineResponse20018Ips {\n");
    
    sb.append("    serviceId: ").append(toIndentedString(serviceId)).append("\n");
    sb.append("    serviceModule: ").append(toIndentedString(serviceModule)).append("\n");
    sb.append("    serviceHostname: ").append(toIndentedString(serviceHostname)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}
